guides/source/action_mailbox_basics.md

```diff
@@ -324,9 +324,11 @@ end
 
 ## Incineration of InboundEmails
 
-By default, an InboundEmail that has been successfully processed will be
-incinerated after 30 days. This ensures you're not holding on to people's data
-willy-nilly after they may have canceled their accounts or deleted their
+By default, an InboundEmail that has been processed will be
+incinerated after 30 days. The InboundEmail is considered as processed
```

Comment:
> The InboundEmail is considered as processed whether the job processing fails or succeeds.

The InboundEmail is considered as processed when its status changes to delivered, failed or bounced.